keylime/tenant.py

```diff
@@ -434,19 +436,18 @@ def check_ek(self, ek, ekcert, tpm):
                 logger.warning(
                     "No EK cert provided, require_ek_cert option in config set to True")
                 return False
-            elif not tpm.verify_ek(base64.b64decode(ekcert), ek):
+            elif not self.tpm_instance.verify_ek(base64.b64decode(ekcert), ek):
```

Comment:
I suggest we move verify_ek (as well as other hardware unrelated implementations) out of tpm, tenant doesn't need a hardware tpm to run.

Reply:
thanks for noticing that, would explain why my quotes were failing.

on the topic of moving verify_ek we should discuss that in an issue.